src/exam/ExamAPIError.jsx

```diff
@@ -1,66 +1,59 @@
-import React, { useState, useContext } from 'react';
-import { Alert, Icon } from '@edx/paragon';
+import React, { useContext } from 'react';
+import { getConfig } from '@edx/frontend-platform';
+import { Alert, Hyperlink, Icon } from '@edx/paragon';
 import { Info } from '@edx/paragon/icons';
-import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { injectIntl, intlShape, FormattedMessage } from '@edx/frontend-platform/i18n';
 import ExamStateContext from '../context';
+import messages from './messages';
 
-export default function ExamAPIError() {
+function ExamAPIError({ intl }) {
   const state = useContext(ExamStateContext);
-  const { apiErrorMsg, proctoringSettings } = state;
-  const { contact_us: contactUs, platform_name: platformName } = proctoringSettings;
-  const [showError, setShowError] = useState(false);
+  const { apiErrorMsg } = state;
 
-  const renderHeading = () => {
-    if (contactUs && platformName) {
+  const shouldShowApiErrorMsg = !!apiErrorMsg && !apiErrorMsg.includes('<');
+
+  const renderSupportMsg = () => {
+    const { SITE_NAME, SUPPORT_URL } = getConfig();
+
+    if (SITE_NAME && SUPPORT_URL) {
       return (
         <FormattedMessage
-          id="exam.apiError.text1"
+          id="exam.apiError.supportText.one"
           defaultMessage={
-            'A system error has occurred with your exam. '
-              + 'Please reach out to {supportLink} for assistance, '
-              + 'and return to the exam once you receive further instructions.'
+            'If the issue persists, please reach out to {supportLink} for assistance, '
+            + 'and return to the exam once you receive further instructions.'
           }
-          values={{ supportLink: <a data-testid="support-link" href={contactUs} target="_blank" rel="noopener noreferrer">{platformName} Support</a> }}
+          values={{
+            supportLink: (
+              <Hyperlink
+                data-testid="support-link"
+                destination={SUPPORT_URL}
+                target="_blank"
+              >
+                {SITE_NAME} Support
+              </Hyperlink>
+            ),
+          }}
         />
       );
     }
 
-    return (
-      <FormattedMessage
-        id="exam.defaultError"
-        defaultMessage="A system error has occurred with your exam. Please reach out to support for assistance."
-      />
-    );
+    return intl.formatMessage(messages.supportTextTwo);
   };
 
   return (
     <Alert variant="danger" data-testid="exam-api-error-component">
       <Icon src={Info} className="alert-icon" />
-      <Alert.Heading data-testid="heading">
-        {renderHeading()}
+      <Alert.Heading data-testid="error-details">
```

Comment:
Why not just include the heading rendering also into the function `renderSupportMsg`? It's going to be easier to read, no?